tests/internal/test_serverless.py

```diff
@@ -120,3 +117,5 @@ def find_spec(self, fullname, *args):
     import ddtrace
     import ddtrace.contrib.aws_lambda  # noqa:F401
     import ddtrace.contrib.psycopg  # noqa:F401
+
+    sys.meta_path = sys.meta_path[1:]
```

Comment:
`try/finally` ?

otherwise could this pollute the state of other jobs if an exception is raised?